go/cmd/dolt/commands/tblcmds/import.go

```diff
@@ -380,10 +380,10 @@ func (cmd ImportCmd) Exec(ctx context.Context, commandStr string, args []string,
 	skipped, verr := mvdata.MoveData(ctx, dEnv, mover, mvOpts)
 
 	if skipped > 0 {
-		cli.PrintErrln(color.YellowString("Lines skipped: %d", skipped))
+		cli.Println(color.YellowString("Lines skipped: %d", skipped))
```

Comment:
This sends text to STDOUT instead of STDERR. Intentional?

Reply:
Yea, that way only the file contents themselves are sent to the stderr descriptor. Thoughts on this?

Reply:
This is the total opposite of what should happen

"main" program output gets printed to STDOUT and auxiliary status info / error messages get printed to STDERR

Are you claiming that MoveData prints table info to STDERR? Because I don't believe you

Reply:
No I'm claiming that bad rows that do not get imported into the database are sent into STDERR. I can also print them in STDOUT but then the user does not have the option to get a neat file piped from STDERR. Ultimately, either solution is fine with me. I think the users benefits more with this approach